servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/AsyncContext.java

```diff
@@ -175,19 +393,77 @@ public static void clear() {
      * @throws NullPointerException (optional behavior) if {@code key} is {@code null} and the underlying
      * {@link AsyncContextMap} implementation doesn't support {@code null} keys or values.
      * @see AsyncContextMap#containsKey(AsyncContextMap.Key)
+     * @deprecated Use {@link #containsKey(ContextMap.Key)}
      */
-    public static boolean containsKey(AsyncContextMap.Key<?> key) {
+    @Deprecated
+    public static boolean containsKey(final AsyncContextMap.Key<?> key) {
         return current().containsKey(key);
     }
 
+    /**
+     * Convenience method to determine if the current context contains a key-value pair corresponding to the
+     * {@code key}.
+     *
+     * @param key The {@link ContextMap.Key} to lookup.
+     * @return {@code true} if the current context contains a key-value pair corresponding to the {@code key},
+     * {@code false} otherwise.
+     * @throws NullPointerException (optional behavior) if {@code key} is {@code null} and the underlying
+     * {@link ContextMap} implementation doesn't support {@code null} keys.
+     * @see ContextMap#containsKey(ContextMap.Key)
+     */
+    public static boolean containsKey(final ContextMap.Key<?> key) {
+        return context().containsKey(key);
+    }
+
+    /**
+     * Convenience method to determine if the current context contains a key-value pair with the specified
+     * {@code value}.
+     *
+     * @param value the value to lookup.
+     * @return {@code true} if this context contains one or more a key-value entries with the specified {@code value},
+     * {@code false} otherwise.
+     * @throws NullPointerException (optional behavior) if {@code value} is {@code null} and the underlying
+     * {@link ContextMap} implementation doesn't support {@code null} values.
+     * @see ContextMap#containsValue(Object)
+     */
+    public static boolean containsValue(final Object value) {
+        return context().containsValue(value);
+    }
+
+    /**
+     * Convenience method to determine if this context contains a key-value pair matching the passed {@code key} and
+     * {@code value}.
+     *
+     * @param key The {@link ContextMap.Key} to lookup.
+     * @param value The value to match.
+     * @param <T> The anticipated type of object associated with the {@code key}.
+     * @return {@code true} if this context contains a key-value pair matching the passed {@code key} and
+     * {@code value}, {@code false} otherwise.
+     * @throws NullPointerException (optional behavior) if {@code key} or {@code value} is {@code null} and the
+     * underlying {@link ContextMap} implementation doesn't support {@code null} keys or values.
+     */
+    public static <T> boolean contains(final ContextMap.Key<T> key, @Nullable final T value) {
+        return context().contains(key, value);
+    }
+
+    /**
+     * Convenience method to determine the number of key/value pairs in the current context.
```

Comment:
Consider "key/value pairs" -> "entries" in the whole class. Java users are familiar with the term.
Same for API, eg. `forEachEntry`.
A matter of taste so no strong feelings.